### DISCUSSION TITLE
kpanda/user-guide/gpu/nvidia/full_gpu_userguide

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2496] [Announcements]
# kpanda/user-guide/gpu/nvidia/full_gpu_userguide

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/gpu/nvidia/full_gpu_userguide.html

**Comment by kebe7jun** [comment]
> 物理卡数量（nvidia.com/vgpu）

写错了吧。

**Reply by lemontrees123** [reply]
已更新
